libkbfs/journal_server.go

```diff
@@ -303,12 +291,19 @@ func (j *JournalServer) Disable(ctx context.Context, tlfID TlfID) (
 			"are any dirty blocks outstanding", tlfID)
 	}
 
-	tlfJournal.shutdown()
-
-	j.log.CDebugf(ctx, "Disabled journal for %s", tlfID)
+	// Disable the journal.  Note that we don't bother deleting the
```

Comment:
Yeah seems to me that this kind of concurrency (both what you wrote and what I wrote) are best solved at a higher layer if it's not acceptable for the concurrent write to go to a server after the journal is re-enabled.  I guess the main reason to keep disabled journals around is just to keep its state around, and to guarantee there aren't two concurrently-live tlfJournals accessing the same data on disk.  (For example, what if some future code keeps a reference to the tlfJournal and decides to re-enable it under certain conditions?  Or if the tlfJournal background goroutine has cleanup tasks to do that aren't directly tied to flushing?)  I'll update this comment to reflect that.